fhir-core/src/main/java/com/ibm/fhir/core/util/ResourceTypeHelper.java

```diff
@@ -207,6 +207,18 @@ private static Set<String> getResourceTypesFor(FHIRVersionParam fhirVersion) {
         }
     }
 
+    /**
+     * Get a list of unique resource type names for all possible FHIRVersionParam values
```

Comment:
might be worth mentioning that only concrete types are returned (not abstract types)

Reply:
added concrete comment to javadoc